test/utility-test.js

```diff
@@ -99,6 +99,30 @@ describe('Rawcrawl Util', function() {
       expect(obj).to.be.an('array');
     });
   });
+});
+
+describe('Database Util', function() {
+  before(function(done) {
+    this.timeout(10000);
+    console.log("Posting test crawls to database...")
+    /* Post two crawls to db for testing */
+    var Crawler = require('../src/lib/crawler.js').Crawler;
+    var _ = require('lodash');
+    var src = require('../src/program');
+    var Promise = require('bluebird');
+    var db_url = 'postgres://postgres:postgres@127.0.0.1:5432/circle_test';
```

Comment:
This is already defined at the top of the file.
You could also move these `require`s to the top of the file.